pkg/rootless/rootless_linux.go

```diff
@@ -501,7 +500,7 @@ func becomeRootInUserNS(pausePid, fileToRead string, fileOutput *os.File) (bool,
 	defer gosignal.Reset()
 	go func() {
 		for s := range c {
-			if s == signal.SIGCHLD || s == signal.SIGPIPE {
```

Comment:
Should we use x/sys/unix instead of syscall? I think unix is more maintained

Reply:
might as well given we are editing, nice change @giuseppe

Reply:
sure.  I've updated the patch to replace syscall with x/sys/unix